applications/TrilinosApplication/custom_python/add_trilinos_processes_to_python.cpp

```diff
@@ -30,6 +31,31 @@ typedef TrilinosSpace<Epetra_FECrsMatrix, Epetra_FEVector> TrilinosSparseSpaceTy
 typedef UblasSpace<double, Matrix, Vector> TrilinosLocalSpaceType;
 typedef LinearSolver<TrilinosSparseSpaceType, TrilinosLocalSpaceType > TrilinosLinearSolverType;
 
+// Helpers to define Trilinos VariationalDistanceCalculatorProcess
+template<unsigned int TDim> using TrilinosVariationalDistanceCalculation = VariationalDistanceCalculationProcess<TDim,TrilinosSparseSpaceType, TrilinosLocalSpaceType, TrilinosLinearSolverType>;
```

Comment:
Just for my understanding, is this a sort of hacky "partial template instantiation" to avoid declaring a 2D and 3D type for the Trilinos variants of the variational distance process?

Reply:
This is just a typedef with a template argument. One could just copy the thing on the right of the `=` everywhere you see this one used, but it is quite long...

Reply:
I've never seen a templated typedef. Good to know. Thanks!